django/test/test_create.py

```diff
@@ -0,0 +1,100 @@
+from . import utils, data
+import requests
+import json
+import pytest
+
+def test_valid_data():
+
+
+    headers = {'Authorization': data.user["asap_access_token"]}
+    params = {'title': 'test_title',
+           'desc': 'test_desc',
+           'max_players': 6,
+           'sport': 'basketball',
+           'start_time': str('2019-12-01 12:00'),
+           'duration': 30,
+           'location_lng': 0.0,
+           'location_lat': 0.0,
+           'location_name': 'test_loc',
+           'comp_level': 1}
+
+    try:
+        res = requests.post("http://localhost:8000/games/host", data=json.dumps(params), headers=headers)
```

Comment:
Use res.json() to turn the res into a Python dict. Then you can assert that a key "game_id" is in the dict response.